signatures/Dilithium/dilithium/bench/skylake-avx2_medium_ref-short-sk.yaml

```diff
@@ -0,0 +1,10 @@
+platform: ref
+comment: 'https://pq-crystals.org/dilithium/data/dilithium-submission-nist-round2.zip'
+param: weak
+impl: skylake-avx2
+timings:
+  unit: cycles
+  gen: 107823
+  enc|sign: 342896
+  comment: median timings (the signing algorithm varies in time)
+  dec|vrfy: 108988
```

Comment:
this is a different paramete rset, isn't it? (different `sk` size)
(comment applies to all of these)

Reply:
No, the difference is that in the short-sk version the sk is stored as a seed and expanded when the signature is created. In the other case the sk is already stored kinda expanded (without A). See Sec. 3.1 in the Dilithium doc. However, I didn't find what parts are exactly stored in the "short sk" case. I guess it's the case where only the seed ρ is stored.

Reply:
Maybe we should create additional parameter sets for that.